js_modules/dagster-ui/packages/ui-core/src/runs/RunsFilterInput.tsx

```diff
@@ -396,6 +401,43 @@ export const useRunsFilterInput = ({tokens, onChange, enabledFilters}: RunsFilte
     },
   });
 
+  const partitionsFilter = useStaticSetFilter({
+    name: 'Partition',
+    icon: 'partition',
+    allValues: partitionValues,
```

Comment:
ah yeah thats a good call, this will certainly increase backend load so would be good to do that instead

Reply:
@prha Do you have any concerns with us loading all of the partition names for an org here? It would be using [this query](https://github.com/dagster-io/dagster/blob/d72f6b60a76cc0f5bd6fbd67b3be09ca02e8251e/js_modules/dagster-ui/packages/ui-core/src/runs/RunsFilterInput.tsx#L622)

Reply:
I'm guessing it might time out 😬